deployments/operator/network-policy/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
```

Comment:
operator README (or some other place?) could mention when these are used

Reply:
These are not used at the moment. I can add a note about using them in the readme.

Reply:
I decided to drop the network policies as I couldn't verify that they worked. In principle they should work, but for me the traffic wasn't stopped when the namespace didn't have the required labels. Maybe it's my CNI (weave).